### PR TITLE
fix: Clear url when link dialog opens in richtext editor

### LINKED ISSUE #800: Story Maps: link UI prefills URL from a previous link
https://user-images.githubusercontent.com/10799815/227548253-0bb000c8-7d66-4285-832c-0946980693e3.mov

## Description
When adding a hyperlink to text in Story Maps, the previous hyperlink is prefilled

## Steps To Reproduce

1. Create a story map
2. Create a chapter
3. Highlight text and insert a URL
4. Highlight a second line of text and click the "Link" button
5. See error

## Expected behavior
The field for inserting the URL should be blank.
image

## Actual behavior
The link from the previous URL is prefilled
image

### PR DESCRIPTION
**Pull request by josebui**
## Description

Clear url when link dialog opens in richtext editor

### Checklist
- [x] Corresponding issue has been opened
- [x] New tests added

### Related Issues
Fixes #800

### CODE REVIEW

**Review by paulschreiber — APPROVED** [review]

**Comment by paulschreiber** [comment]
Suggestion: add a test / updated an existing test to confirm the field is empty when the dialog opens.

**Review by paulschreiber — APPROVED** [review]

File: src/common/components/RichTextEditor/index.test.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+    await act(async () => {
+      fireEvent.mouseDown(linkButton);
+    });
+
+    const dialog = screen.getByRole('dialog', { name: 'Add Link' });
+
+    const urlInput = within(dialog).getByRole('textbox', { name: 'Link' });
+    await act(async () => {
+      fireEvent.change(urlInput, { target: { value: 'test.com' } });
```
Comment (paulschreiber) [inline src/common/components/RichTextEditor/index.test.js:65]:
```suggestion
      fireEvent.change(urlInput, { target: { value: 'example.com' } });
```

File: src/common/components/RichTextEditor/index.test.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      initialFocused
+      value={[
+        {
+          type: 'paragraph',
+          children: [
+            {
+              text: 'asdasd asd asd ',
+            },
+            {
+              url: 'https://test.com',
```
[the remark this answers is not in the document]
Comment (paulschreiber) [reply src/common/components/RichTextEditor/index.test.js:31]: example.com is a domain reserved for this purpose. In our case, we're not making a HTTP request, since this is mocked, but it's a good practice.
